dev/com.ibm.ws.messaging.common/src/com/ibm/ws/messaging/lifecycle/SingletonMonitor.java

```diff
@@ -121,23 +145,93 @@ public synchronized void modified(Map<String, Object> properties) {
             if (removed.size() > 0) debug(this, tc, "Singleton declarations removed:", removed);
             if (added.size() > 0) debug(this, tc, "Singleton declarations added:", added);
         }
-        if (isAnyTracingEnabled() && tc.isDebugEnabled()) Tr.exit(this, tc, "modified");
+        if (isAnyTracingEnabled() && tc.isEntryEnabled()) exit(this, tc, "modified");
     }
 
     @Deactivate
     public void deactivate() {
+        // this is only here to allow debug
         if (isAnyTracingEnabled() && tc.isDebugEnabled()) debug(tc, this + " deactivate");
```

Comment:
okay, so this line isn't changed, but shouldn't this call be `debug(this, tc, "deactivate")` rather than `debug(tc, this + " deactivate")` ?